doc/decisions/3_in_review/change_tracking.md

```diff
@@ -52,9 +52,9 @@ The library `libelektra-core` must be kept minimal.
 
 - It is possible to do change tracking with reasonable memory and computation overhead
 - It is possible to design a single change tracking API that is useful for all existing and future plugins
-- False positivies are okay
+- False positives are okay
 
-  - this may happend when some keys have been changed by the user, but have subsequentially been "unchanged" by a transformation plugin
+  - this may happened when some keys have been changed by the user, but have subsequentially been "unchanged" by a transformation plugin
```

Comment:
```suggestion
  - this may happens when some keys have been changed by the user, but have subsequentially been "unchanged" by a transformation plugin
```